src/main/resources/component-biz.conf

```diff
@@ -1283,6 +1283,7 @@ security {
         fallback-language = "en"
         known-languages = ["de", "en", "fr", "nl", "it", "es", "pl", "cs", "hu", "da", "sv", "ro", "ru", "sk", "bg"]
         display-languages = ["de", "en", "fr", "nl", "it", "es", "pl", "cs", "hu", "da", "sv", "ro", "ru", "sk", "bg"]
+        defaultRoles = ["permission-view-files", "permission-view-processes", "permission-view-audit-log"]
```

Comment:
please doublecheck that these are not granted to _not logged in_ users

Reply:
Shouldn't we grant them using a profile via logged-in {...} using the same approach in a product, these permissions can be revoked again

Reply:
You mean like this? I'm not sure im using the profile correctly